### DISCUSSION TITLE
How to add a comment that is designated Internal

### DISCUSSION THREAD

**Discussion by Rastlinux** [ankitpokhrel/jira-cli#685] [Q&A] [answered]
When i add comments they are automatically sent to the client. However, sometimes we create comments that are set to internal where the comment is not sent out and is not visible by the end user. We use this to put in code or other items that would confuse the user  but we can later look up in a search.

**Comment by ankitpokhrel** [comment]
Hi @Rastlinux, thank you for explaining your usecase.

> we create comments that are set to internal 

Could you please help me understand how do you mark the comment as internal? Is it some Jira config enabled on your instance or is this based on the role assigned to the jira users?

**Reply by dynek** [reply, marked as answer]
See: https://github.com/ankitpokhrel/jira-cli/pull/825

**Comment by ankitpokhrel** [comment]
Done in #825
